<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDialog</class>
 <widget class="QDialog" name="CalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>194</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>648</width>
    <height>194</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>648</width>
    <height>194</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#2c3e50;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Tracked Hand Diameter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true"> QPushButton {
     background-color: #ecf0f1;
     border-style: outset;
     border-width: 2px;
     border-color: #2c3e50;
     font: bold 12px;
     min-width: 10em;
     padding: 6px;
 }
 QPushButton:hover {
     background-color: #bdc3c7;
     border-style: inset;
 }
 QPushButton:pressed {
     background-color: #95a5a6;
     border-style: inset;
 }</string>
          </property>
          <property name="text">
           <string>Save Offset</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Reference Ball Diameter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit">
          <property name="styleSheet">
           <string notr="true"> QLineEdit {
     border: 0px solid gray;
     padding: 10px;
     background: #34495e;
     selection-background-color: #34495e;
	color: #f1c40f;
 }
</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="styleSheet">
           <string notr="true"> QLineEdit {
     border: 0px solid gray;
     padding: 10px;
     background: #34495e;
     selection-background-color: #34495e;
	color: #f1c40f;
 }
</string>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_3">
          <property name="styleSheet">
           <string notr="true"> QLineEdit {
     border: 0px solid gray;
     padding: 10px;
     background: #bdc3c7;
     selection-background-color: #34495e;
 }
</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">QLabel{
	background: #2c3e50;
	padding: 10px;
	color: white;
}</string>
          </property>
          <property name="text">
           <string>Calculated Offset</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
